packages/python-uncompyle6/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname=python-uncompyle6
+pkgver=1996.616e5c8
+pkgrel=1
+pkgdesc='A Python cross-version decompiler.'
+groups=('blackarch' 'blackarch-decompiler')
+arch=('any')
+url='https://github.com/rocky/python-uncompyle6'
+license=('GPL3')
+depends=('python2' 'python2-hypothesis' 'python2-pytest')
+makedepends=('git' 'python2')
+source=('git+https://github.com/rocky/python-uncompyle6.git')
+sha512sums=('SKIP')
+
+pkgver() {
+  cd python-uncompyle6
+  
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
+  cd python-uncompyle6
+  
+  python2 setup.py build
+}
+
+package() {
+  cd python-uncompyle6
+  
+  python2 setup.py install --prefix=/usr --root="$pkgdir" --optimize=1
+  
+  install -Dm 644 -t "$pkgdir/usr/share/doc/python-uncompyle6/" README.rst
+  install -Dm 644 COPYING "$pkgdir/usr/share/licenses/python-uncompyle6/COPYING"
```

Comment:
Use this instead:
```
"$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```

It's always good practice to use `$pkgname` instead of hardcoding it and the license file should be named `LICENSE`, not `COPYING`.

Reply:
Complete.